http4s/src/main/scala/geotrellis/server/http4s/maml/ExampleMamlService.scala

```diff
@@ -0,0 +1,90 @@
+package geotrellis.server.http4s.maml
```

Comment:
looking towards keeping this example into the branch, mb it makes sense to move it into `examples` package?